SINGLE_CONNECTION.md

```diff
@@ -0,0 +1,195 @@
+
+The following is an example of creating a single connection against a **NuoDB** database using `callback` semantics. 
+
+```js
+let rows;
+const driver = new Driver();
+
+driver.connect(config, (err, connection) => {
+   if (err) {/* handle connection error */}
+
+   connection.execute("SELECT * FROM SYSTEM.NODES;", (err, results) => {
+      if (err) {/* handle execution error */}
+
+      results.getRows((err, rows) => {
+         if (err) {/* handle getting rows error */}
+
+         // use rows
+      });
+
+      results.close(err => {
+         if (err) {/* handle result set closure error */}
+      })
+   });
+   
+   connection.close((err) => {
+      if (err) {/* handle closure error */}
+   });
+});
+```
+
+## Using Promises 
+
+The following is an example of creating a single connection against a **NuoDB** database using `Promise` semantics. 
+
+```js
+let rows;
+const driver = new Driver();
+
+driver.connect(config)
+   .then(connection => {
+
+      connection.execute("SELECT * FROM SYSTEM.NODES;")
+         .then(results => {
+            
+            results.getRows()
+               .then(rows => { /* use rows */ })
+               .catch(err => { /* handle getting rows error */ });
+            
+            results.close()
+               .catch(err => { /* handle results closure error */ })
+         })
+         .catch(err => { /* handle SQL execution error */ });
+      
+      connection.close()
+         .catch(err => { /* handle connection closure error */ })
+   
+   })
+   .catch(err => { /* handle connection error */ }) 
+```
+
+
+
+## Connection Properties
+
+Key	| Value
+---|---
+allowSRPFallback=[ true \| false ] | Allows the driver to fall back to SRP if the TLS connection fails (when used in combination with trustStore ). Required (set to true) during rolling upgrade of the security protocol.
+ciphers=cipherlist |  Enables the client to provide one or more appropriate ciphers for encrypting the network connection to the database. If this property is not specified the default ciphers AES-256-CTR and RC4 are enabled. The server chooses a preferred cipher from this list. If no matching cipher is found, the connection will be refused. The cipherlist is comma-separated and cipher names are case-sensitive and must be provided exactly as shown. The full list of available ciphers are AES-256-CTR, RC4, and None.
+clientInfo=info | Arbitrary information about the connecting client. This is available in the CLIENTINFO column of the SYSTEM.CONNECTIONS and SYSTEM.LOCALCONNECTIONS system tables.
+clientProcessID=id | The process ID (pid) of the connecting client. The client may set this property only if not already set by the driver. The .NET and Python drivers set this for the client. The client setting of this property does not overrides the driver setting of this property. The clientProcessId is then available in the CLIENTPROCESSID column of the SYSTEM.CONNECTIONS and SYSTEM.LOCALCONNECTIONS system tables.
```

Comment:
I think "The .NET and Python drivers set this for the client." could be removed. We don't even rlly have a .NET driver.